docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+scalardl-auditor-envoy           LoadBalancer   10.111.141.43    <pending>     40051:31553/TCP,40052:31171/TCP   99s
+scalardl-auditor-envoy-metrics   ClusterIP      10.104.245.188   <none>        9001/TCP                          99s
+scalardl-auditor-headless        ClusterIP      None             <none>        40051/TCP,40053/TCP,40052/TCP     99s
+scalardl-auditor-metrics         ClusterIP      10.105.119.158   <none>        8080/TCP                          99s
+scalardl-ledger-envoy            LoadBalancer   10.96.239.167    <pending>     50051:32714/TCP,50052:30857/TCP   109s
+scalardl-ledger-envoy-metrics    ClusterIP      10.97.204.18     <none>        9001/TCP                          109s
+scalardl-ledger-headless         ClusterIP      None             <none>        50051/TCP,50053/TCP,50052/TCP     109s
+scalardl-ledger-metrics          ClusterIP      10.104.216.189   <none>        8080/TCP                          109s
+```
+
+### Pause
+
+You can send the pause request to Scalar product pods on the Kubernetes environment.
+
+* Example
+  * ScalarDB Server
+    ```console
+    kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c pause -s _scalardb._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Ledger
+    ```console
+    kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c pause -s _scalardl-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Auditor
+    ```console
+    kubectl run scalar-admin-pause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c pause -s _scalardl-auditor-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+
+### Unpause
+
+You can send the unpause request to Scalar product pods on the Kubernetes environment.
+
+* Example
+  * ScalarDB Server
+    ```console
+    kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c unpause -s _scalardb._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Ledger
+    ```console
+    kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c unpause -s _scalardl-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+  * ScalarDL Auditor
+    ```console
+    kubectl run scalar-admin-unpause --image=ghcr.io/scalar-labs/scalar-admin:<tag> --restart=Never -it -- -c unpause -s _scalardl-auditor-admin._tcp.<helm release name>-headless.<namespace>.svc.cluster.local
+    ```
+
+### Check the pause completed time and unpause started time
+
+The scalar-admin pods output the **pause completed time** and **unpause started time** to the stdout. You can also see those times using the `kubectl logs` command.
```

Comment:
```suggestion
The `scalar-admin` pods output the pause completed time and unpause starting time to stdout. You can also see those times by running the `kubectl logs` command.
```

Reply:
As I mentioned in another comment, I would like to discuss whether `unpause started` or `unpause starting` is better.